db/seeds.rb

```diff
@@ -6,31 +6,68 @@
-Language.find_or_create_by({id:1,name:'English',code:'EN'})
-Language.find_or_create_by({id:2,name:'Español',code:'ES'})
-Language.find_or_create_by({id:3,name:'Français',code:'FR'})
+
+Language.find_or_create_by(id: 1, name: 'English', code: 'EN')
+Language.find_or_create_by(id: 2, name: 'Español', code: 'ES')
+Language.find_or_create_by(id: 3, name: 'Français', code: 'FR')
 Language.connection.execute("select setval('languages_id_seq', (select max(id) from languages))")
 
 
-# note, these hardcoded id's are needed to match the migrated data
-Currency.find_or_create_by({id:1,name:'Argentinean Peso',code:'ARS',symbol:'AR$'})
-Currency.find_or_create_by({id:2,name:'U.S. Dollar',code:'USD',symbol:'US$'})
-Currency.find_or_create_by({id:3,name:'British Pound',code:'GBP',symbol:'GB£'})
-Currency.find_or_create_by({id:4,name:'Nicaraguan Cordoba',code:'NIO',symbol:'NI$'})
+Currency.find_or_create_by(id: 1, name: 'Argentinean Peso', code: 'ARS', symbol: 'AR$')
+Currency.find_or_create_by(id: 2, name: 'U.S. Dollar', code: 'USD', symbol: 'US$')
+Currency.find_or_create_by(id: 3, name: 'British Pound', code: 'GBP', symbol: 'GB£')
+Currency.find_or_create_by(id: 4, name: 'Nicaraguan Cordoba', code: 'NIO', symbol: 'NI$')
 Currency.connection.execute("select setval('currencies_id_seq', (select max(id) from currencies))")
 
 
-# note, these hardcoded id's are needed to match the migrated data
-Country.find_or_create_by({id:1,name:'Argentina',iso_code:'AR',default_language_id:2,default_currency_id:1})
-Country.find_or_create_by({id:2,name:'Nicaragua',iso_code:'NI',default_language_id:2,default_currency_id:4})
-Country.find_or_create_by({id:3,name:'United States',iso_code:'US',default_language_id:1,default_currency_id:2})
+Country.find_or_create_by(id: 1, name: 'Argentina', iso_code: 'AR', default_language_id: 2, default_currency_id: 1)
+Country.find_or_create_by(id: 2, name: 'Nicaragua', iso_code: 'NI', default_language_id: 2, default_currency_id: 4)
+Country.find_or_create_by(id: 3, name: 'United States', iso_code: 'US', default_language_id: 1, default_currency_id: 2)
 Country.connection.execute("select setval('countries_id_seq', (select max(id) from countries))")
 
+
 # for now mapping the '0' Person refs to 'null' and allowing null refs in the schema
 # Person.find_or_create_by({id:0, name: 'dummy', first_name: 'dummy', division_id: Division.root_id})
+
+# handy temporary change for debugging
+# Option.destroy_all
+# OptionSet.destroy_all
+
+loan_status = OptionSet.find_or_create_by(division: Division.root, model_type: Loan.name, model_attribute: 'status')
+# beware rerunning these different position values, will result in duplicated data
+# a more verbose approach would be needed if that is important to support
+# also we can consider dropping the 'position' assignments until an ordering different than 'value' is needed
+loan_status.options.find_or_create_by(value: 1, position: 1).set_label_list(en: 'Active', es: 'Prestamo Activo')
```

Comment:
my understanding is that the set of options are potentially customized per division.  we can potentially use an 'internal_name' style string instead of an id for the 'value'.  when need to think about the implications of that when designing the ability for end users to customize these lists.  
@linesarefuzzy , can you weigh in here with how you want to model this?